common/constants.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+KUBE_API_ROOT = os.environ.get('KUBE_API_ROOT', 'http://localhost:8080/api/v1/')
```

Comment:
These should all have comments explaining what they mean / configure.

Things that are part of "groups" (e.g. logging constants, DEFAULT_PROFILE_*, annotation keys) should be grouped accordingly using newlines and comments to explain the block.